ob-pattern-1/templates/obkm/wso2ob-pattern-1-obkm-deployment.yaml

```diff
@@ -127,3 +127,4 @@ spec:
       - name: wso2ob-pattern-1-obkm-deployment-jaggeryapps-ccportal
         configMap:
           name: wso2ob-pattern-1-obkm-deployment-jaggeryapps-ccportal
+          
```

Comment:
Missing newline

Reply:
Fixed in https://github.com/wso2/kubernetes-open-banking/pull/22/commits/801acbc3a7226e2aa43f6fbdeffb5f723f740695